### ISSUE TITLE
Fine tune model

### ISSUE THREAD

**Issue by gauravlochab** [sachinraja13/TabStructNet#7]
Could you tell the best way to fine tune the model on my custom dataset at different training levels ?
'Finetune layers from ResNet stage 4 and up'
'Fine tune all layers'
these comments in tabnet.py are unclear
Thanks for the amazing work appreciate your prompt reply!

**Comment by sachinraja13** [comment]
Hi,

I would recommend training heads for 20 epochs, training ResNet stage 4 and up for another 20 epochs, and training all layers for additional 40 epochs. You can have a look at how loss is decreasing and can appropriately change learning rate in the config.py file.

Please make sure to execute "python setup.py install" after you have made any changes to the model.py or config.py file in the mrcnn folder.

Hope this helps.

**Comment by gauravlochab** [comment]
thanks, it helped

**Comment by prati2a22** [comment]
> thanks, it helped

Hey @gauravlochab, can you share how you annotated the data for finetuning? Which specific tool you used for annotation? 
I keep getting a KeyError for 'start_row' as my annotator tool did not generate the start_row, start_col and end_row, end_col fields.

**Comment by gauravlochab** [comment]
Hi @prati2a22 , i cant share the data with but you can easily annotate the data as shown in the tabstructnet paper. i used cvat for annotating the images.

**Comment by prati2a22** [comment]
> Hi @prati2a22 , i cant share the data with but you can easily annotate the data as shown in the tabstructnet paper. i used cvat for annotating the images.

Hey thanks for the info @gauravlochab, I checked out the CVAT documentation and I was wondering how did you add the "start_row", "start_col", "end_row", "end_col" fields. That's the part I'm struggling with.
